features/fixtures/ios-swift-cocoapods/iOSTestApp/ViewController.swift

```diff
@@ -42,6 +42,18 @@ class ViewController: UIViewController {
     @IBAction func clearUserData(_ sender: Any) {
         NSLog("Clear user defaults")
         UserDefaults.standard.removePersistentDomain(forName: Bundle.main.bundleIdentifier!)
+        do {
```

Comment:
I think this is comment of a single line comment: "Delete any existing ..?..".  OOM files, error files, session files, all of the above?